crates/dialogs/README.md

```diff
@@ -20,8 +19,12 @@
   <sub>Built with 🦀🕸 by <a href="https://rustwasm.github.io/">The Rust and WebAssembly Working Group</a></sub>
 </div>
 
-This crate provides wrapper for `alert`, `prompt` and `confirm` functions.
+This crate provides wrapper for the following functions.
+- [`alert`](https://developer.mozilla.org/en-US/docs/Web/API/Window/alert)
+- [`confirm`](https://developer.mozilla.org/en-US/docs/Web/API/Window/confirm)
+- [`prompt`](https://developer.mozilla.org/en-US/docs/Web/API/Window/prompt)
+
 `web-sys` provides a raw API which is hard to use. This crate provides an easy-to-use,
 idiomatic Rust API for these functions.
 
-See the documentation for [`alert`], [`prompt`] and [`confirm`] for more information.
+See the API documentation to learn more.
```

Comment:
Would it make sense for "API documentation" to be a link to the docs here? For ease of use for readers?